packages/app-mobile/components/screens/ConfigScreen/plugins/SearchPlugins.tsx

```diff
@@ -75,41 +97,62 @@ const PluginSearch: React.FC<Props> = props => {
 				installState,
 			};
 		});
-	}, [searchResultManifests, installingPluginsIds, pluginSettings]);
-
-	const onPluginSettingsChange = useCallback((event: OnPluginSettingChangeEvent) => {
-		props.onUpdatePluginStates(event.value);
-	}, [props.onUpdatePluginStates]);
+	}, [searchResultManifests, props.installingPluginIds, pluginSettings]);
 
-	const installPlugin = useInstallHandler(
-		setInstallingPluginIds, pluginSettings, props.repoApi, onPluginSettingsChange, false,
-	);
 
+	const onInstall = props.callbacks.onInstall;
 	const renderResult = useCallback(({ item }: { item: SearchResultRecord }) => {
 		const manifest = item.item.manifest;
 
-		return (
-			<PluginBox
-				themeId={props.themeId}
-				key={manifest.id}
-				item={item.item}
-				installState={item.installState}
-				isCompatible={PluginService.instance().isCompatible(manifest)}
-				onInstall={installPlugin}
-				onAboutPress={openWebsiteForPlugin}
-			/>
-		);
-	}, [installPlugin, props.themeId]);
+		if (item.installState === InstallState.Installed && PluginService.instance().isPluginLoaded(manifest.id)) {
+			return (
+				<InstalledPluginBox
+					pluginId={manifest.id}
+					themeId={props.themeId}
+					pluginSettings={props.pluginSettings}
+					updatablePluginIds={props.updatablePluginIds}
+					updatingPluginIds={props.updatingPluginIds}
+					showInstalledChip={true}
+					callbacks={props.callbacks}
+					onShowPluginInfo={props.onShowPluginInfo}
+				/>
+			);
+		} else {
+			return (
+				<PluginBox
+					themeId={props.themeId}
+					key={manifest.id}
+					item={item.item}
+					installState={item.installState}
+					showInstalledChip={false}
+					isCompatible={PluginService.instance().isCompatible(manifest)}
+					onInstall={onInstall}
+					onAboutPress={openWebsiteForPlugin}
+				/>
+			);
+		}
+	}, [onInstall, props.themeId, props.pluginSettings, props.updatingPluginIds, props.updatablePluginIds, props.onShowPluginInfo, props.callbacks]);
+
+	const renderResultsCount = () => {
+		if (!searchQuery.length) return null;
+
+		return <Text style={styles.resultsCounter} variant='labelLarge'>
+			{_('Results (%d):', searchResults.length)}
+		</Text>;
+	};
 
 	return (
-		<View style={{ flexDirection: 'column' }}>
-			<Searchbar
+		<View style={styles.container}>
+			<TextInput
 				testID='searchbar'
-				placeholder={_('Search')}
+				mode='outlined'
+				left={<TextInput.Icon icon='magnify' />}
```

Comment:
This currently causes a warning when the component is mounted:
```
Warning: TextInput.Icon: Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.
```

[This upstream pull request](https://github.com/callstack/react-native-paper/pull/4385), when merged should remove the warning.